neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -27,6 +27,8 @@ public class SetFilter { public BloomFilter Filter; }
         private bool verack = false;
         private BloomFilter bloom_filter;
 
+        private static readonly Random random = new Random();
```

Comment:
Random class could not be static

Reply:
Could you proceed with the change accordingly?